NAMESPACE

```diff
@@ -22,8 +23,22 @@ exportMethods(make_row_df)
 exportMethods(matrix_form)
 exportMethods(prov_footer)
 exportMethods(subtitles)
+exportMethods(toString)
 import(formatters)
 import(methods)
 import(tibble)
+importFrom(grDevices,dev.off)
+importFrom(grDevices,pdf)
+importFrom(grid,convertHeight)
+importFrom(grid,convertWidth)
+importFrom(grid,get.gpar)
+importFrom(grid,gpar)
+importFrom(grid,grid.draw)
+importFrom(grid,grid.newpage)
+importFrom(grid,plotViewport)
+importFrom(grid,pushViewport)
+importFrom(grid,textGrob)
+importFrom(grid,unit)
+importFrom(stats,na.omit)
 importFrom(utils,head)
```

Comment:
Imported but not in DESCRIPTION file

Reply:
good catch! thanks man. I think we will need to rtables as well